### PR TITLE
implemented py-person-class

### PR DESCRIPTION
**Pull request by mariiahorbova**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,29 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int, **kwargs) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
```
Comment (LiudmylaKulinchenko) [inline app/main.py:7]: You should access class attribute through class name not self

File: app/main.py [hunk 2]
```diff
@@ -1,8 +1,29 @@
...
+        age = person["age"]
+        new_person = Person(name, age)
+        person_list.append(new_person)
+
+    for person in people:
+        name = person["name"]
+        wife = person.get("wife")
+        husband = person.get("husband")
+
+        if wife is not None:
```
Comment (LiudmylaKulinchenko) [inline app/main.py:23]: `is not None` redundant here, since in your case if statement will be True if key exists

File: app/main.py [hunk 3]
```diff
@@ -1,8 +1,29 @@
...
+            Person.people[name].wife = Person.people[wife]
+
+        if husband is not None:
```
Comment (LiudmylaKulinchenko) [inline app/main.py:26]: Same here about `is not None`

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 2 above)
Comment (mariiahorbova) [reply app/main.py]: done

File: app/main.py (hunk 3 above)
Comment (mariiahorbova) [reply app/main.py]: done

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 4]
```diff
@@ -1,8 +1,29 @@
...
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for person in people:
```
[the remark this answers is not in the document]
Comment (mariiahorbova) [reply app/main.py]: done

**Pull request closed without merging**